src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -105,7 +105,13 @@ protected function _tableUnmodified($db)
  */
     protected function _hash(ConnectionInterface $db)
     {
-        $driver = $db->getDriver();
+        if (method_exists($db, 'getDriver')) {
```

Comment:
Method exists check aren't necessary, just change `$driver = $db->getDriver();` to `$driver = $db->driver();`.